packages/app/src/request.ts

```diff
@@ -34,9 +34,7 @@ export const getProtocol = (req: Request): Protocol => {
 
 export const getHostname = (req: Request): string | undefined => {
   let host: string = req.get('X-Forwarded-Host') as string
-
-  if (!host || !trustRemoteAddress(req)) host = req.get('Host') as string
```

Comment:
why did you remove it? trustRemoteAddress may return false, if remote address is different from Host header

Reply:
`trustRemoteAddress` will always return a not null value

https://github.com/tinyhttp/tinyhttp/blob/7bf29536a0e5574a82768d87a0b57a34bf217fa5/packages/proxy-addr/src/index.ts#L88